ios-base/Helpers/Constants.swift

```diff
@@ -27,3 +28,26 @@ enum ErrorDomain: String {
   case generic = "GenericError"
   case parsing = "ParsingError"
 }
+
+struct UI {
```

Comment:
nitpick: Please move this to its own file. This file `Constants` seems to be growing indefinitely with all sort of things.